service/telemetry.go

```diff
@@ -54,19 +51,21 @@ var (
 type telemetryInitializer struct {
 	views      []*view.View
 	ocRegistry *ocmetric.Registry
-	mp         metric.MeterProvider
-	tp         trace.TracerProvider
+	mp         *sdkmetric.MeterProvider
+	tp         *sdktrace.TracerProvider
 	servers    []*http.Server
 
+	serverWG sync.WaitGroup
+
 	useOtel                bool
 	disableHighCardinality bool
 	extendedConfig         bool
 }
 
 func newColTelemetry(useOtel bool, disableHighCardinality bool, extendedConfig bool) *telemetryInitializer {
 	return &telemetryInitializer{
-		mp:                     noopmetric.NewMeterProvider(),
-		tp:                     nooptrace.NewTracerProvider(),
+		mp:                     sdkmetric.NewMeterProvider(),
+		tp:                     sdktrace.NewTracerProvider(),
```

Comment:
Why this change?

Reply:
I wrote the reasoning here in the description, under point number 2. Let me know if any more information would be helpful.